proposals/container-build-plan-spec.md

```diff
@@ -0,0 +1,176 @@
+# Container Build Plan Specification
+
+Specification for building a container image.
+
+Although looking similar, the structure and semantics of similary named properties are different from the Docker/[OCI Image Configuration](https://github.com/opencontainers/image-spec/blob/master/config.md).
+
+## Example
+
+```
+{
+  "baseImage": "gcr.io/distroless/java@sha256:b715126ebd36e5d5c2fd730f46a5b3c3b760e82dc18dffff7f5498d0151137c9",
+  "architecture": "amd64",
+  "os": "linux",
+  "format": "Docker",
+  "created": "2011-12-03T22:42:05Z",
+
+  "config": {
+    "env": {
+      "KEY": "value",
+      "PATH": "/usr/sbin:/usr/bin:/sbin:/bin",
```

Comment:
should we allow passthrough from the baseImage env?
Could path be "PATH:$PATH" ?

Reply:
As discussed, we will consider such extension in the future.